guide/payments/receive.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: guide
+title: Receiving bitcoin
+description: An introduction to how receiving bitcoin works.
+nav_order: 3
+parent: Payments
+permalink: /guide/payments/receive/
+main_classes: -no-top-padding
+# image: /assets/images/guide/payments/receive/header.svg
+---
+
+# Receiving bitcoin
+
+<!--
+
+Editor's notes
+
+This page should cover what to do when receiving bitcoin, how to share and copy addresses etc.
+
+- What addresses to support (native segwit by default)
+- How to present QR codes (allow for specifying an amount)
+- Generating a new address 
+
+-->
+
+The details required by a sender to make a payment, at minimum, is an address. You can think of an address just as you would an invoice, as it is only used once. The receiver generates an address in their Bitcoin application and gives it to the sender. Just as with traditional invoices, a new address needs to be created each time the receiver wants to receive funds.
```

Comment:
This is not accurate, as addresses can be used multiple times. I'd differentiate between "can" and "should".

Reply:
Privacy by default. But I agree perhaps `should` is a better choice of words, still, we need to have strong opinions and recommendations when it comes to these things.

Reply:
fair points @GBKS @pavlenex , thoughts on this suggestion by @danielnordh?

```suggestion
For someone to receive bitcoin, the sender needs to know the recipient’s address. The receiver generates an address in their Bitcoin application and shares it with the sender, either as plain text, a QR code, or a payment link.  You can think of an address just as you would an invoice, you generate a new one for each transaction, as addresses should only be used once to increase privacy. 
```

Reply:
minimal change. 

```suggestion
The details required by a sender to make a payment, at minimum, is an address. You can think of an address just as you would an invoice, as it should only be used once. The receiver generates an address in their Bitcoin application and gives it to the sender. Just as with traditional invoices, a new address needs to be created each time the receiver wants to receive funds.
```

Reply:
I like Daniel's as it is written from the perspective of a receiver, and this is the receive page.

Reply:
True. Good point.